docs/guides/applications/configuration-management/terraform/deploy-a-wordpress-site-using-terraform-and-linode-stackscripts/index.md

```diff
@@ -186,7 +222,7 @@ resource "linode_instance" "my_wordpress_linode" {
 
     -   The `root_pass` argument is assigned to the value of the `random_string` resource previously declared.
 
-    -   To use an existing StackScript you must use the `stackscript_id` argument and provide a valid ID as a value. Every StackScript is assigned a unique ID upon creation. This guide uses the [WordPress on Ubuntu 16.04](https://www.linode.com/stackscripts/view/81736) StackScript provided by Linode user [hmorris](https://www.linode.com/stackscripts/profile/hmorris). This StackScript's ID will be assigned to a Terraform variable later in this guide.
+    -   To use an existing StackScript you must use the `stackscript_id` argument and provide a valid ID as a value. Every StackScript is assigned a unique ID upon creation. This guide uses the [WordPress on Ubuntu 20.04](https://www.linode.com/stackscripts/view/998743) StackScript adapted by the Linode user [hmorris](https://www.linode.com/stackscripts/profile/hmorris). This StackScript's ID will be assigned to a Terraform variable later in this guide.
```

Comment:
That link to the StackScript didn't work for me. Here's one that did: https://cloud.linode.com/stackscripts/998743

Reply:
Fixed this

Reply:
Just popped in a commit with the new links. AFAIK StackScripts are only visible in Cloud - not on the website (at least that i could find).